assets/files.wxs

```diff
@@ -1930,6 +1930,11 @@
             <File Id="filFF877991123D49A58FD15CB83AD44BA7" KeyPath="yes" Source="$(env.ProductSourcePath)\Modules\ThreadJob\ThreadJob.dll" />
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Validate New-User.Can create New-User with explicit properties in local machine
```powershell
Expected exactly 'sn', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1: line 58
58:             $result.SurName | Should -BeExactly "sn"
```